src/libs/auth.js

```diff
@@ -446,7 +446,9 @@ workspaceStore.subscribe((newState, oldState) => {
 authStore.subscribe(
   withErrorReporting('Error loading azure preview group membership', async (state, oldState) => {
     if (becameRegistered(oldState, state)) {
-      const isGroupMember = await Ajax().Groups.group(getConfig().azurePreviewGroup).isMember();
+      // Note that `azurePreviewGroup` is set to true in all non-prod config files, so only call Sam group for prod.
+      const bypassSam = getConfig().azurePreviewGroup !== true;
+      const isGroupMember = bypassSam || (await Ajax().Groups.group(getConfig().azurePreviewGroup).isMember());
```

Comment:
BTW, I verified that `await Ajax().Groups.group(getConfig().azurePreviewGroup).isMember())` will return false if `getConfig().azurePreviewGroup` happens to be `false`.